exercises/concept/lasagna/src/test/java/LasagnaTest.java

```diff
@@ -1,42 +1,90 @@
 import org.junit.Test;
-import org.junit.Ignore;
+
+import utils.Lasagna;
 
 import static org.assertj.core.api.Assertions.*;
 
 public class LasagnaTest {
-    
+
+    @Test
+    public void implemented_expected_minutes_in_oven() {
+        assertThat(new Lasagna().hasMethod("expectedMinutesInOven"))
+                .withFailMessage("Method expectedMinutesInOven must be created")
+                .isTrue();
+        assertThat(new Lasagna().isMethodPublic("expectedMinutesInOven"))
+                .withFailMessage("Method expectedMinutesInOven must be public")
+                .isTrue();
+        assertThat(new Lasagna().isMethodReturnType(int.class, "expectedMinutesInOven"))
+                .withFailMessage("Method expectedMinutesInOven must return an int")
+                .isTrue();
+    }
+
     @Test
     public void expected_minutes_in_oven() {
         assertThat(new Lasagna().expectedMinutesInOven()).isEqualTo(40);
     }
 
-    @Ignore("Remove to run test")
+    @Test
```

Comment:
I believe the test runner actually removes the `@Ignore` annotation when it runs the tests on the Exercism server. The `@Ignore` annotation is normally placed on every test except the first one for local development purposes, i.e. `exercism download --exercise=lasagna --track=java` with the command line tool.

Reply:
Interesting, I didn't know that, but it makes a lot of sense now